main/php_network.h

```diff
@@ -229,7 +232,7 @@ PHPAPI void php_network_freeaddresses(struct sockaddr **sal);
 
 PHPAPI php_socket_t php_network_connect_socket_to_host(const char *host, unsigned short port,
 		int socktype, int asynchronous, struct timeval *timeout, char **error_string,
-		int *error_code, char *bindto, unsigned short bindport 
+		int *error_code, char *bindto, unsigned short bindport, long sockopts
 		TSRMLS_DC);
```

Comment:
Hmm ... you're right. The only occurrence in the 5.5+ codebase is patched in this commit but I guess we can't be sure how/if third-party extensions are using it. I hadn't considered that. I assume this limits the functionality to 5.7/7?